.github/workflows/ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:  
       matrix: 
         include:
-          - os: ubuntu-16.04
+          - os: ubuntu-20.04
```

Comment:
The CI should build against ubuntu-16.04, because this will become the minimum supported version for the static builds of the release. If you raise this, the static builds will not run on older ubuntu version.